src/rotation.zig

```diff
@@ -65,10 +66,27 @@ pub const RotationMode = struct { // MARK: RotationMode
 			const boxTMin = @reduce(.Max, @min(t1, t2));
 			const boxTMax = @reduce(.Min, @max(t1, t2));
 			if(boxTMin <= boxTMax and boxTMax > 0) {
+				var face = Neighbor.dirUp;
```

Comment:
Should be initialized to undefined since you overwrite it in every reachable path.